tests/e2e/e2e.go

```diff
@@ -143,12 +143,12 @@ func setupSuite() {
 
 	// By using default storage class in GKE/EKS (aws), network attached storage
 	// which be used and we must clean them later.
-	// We set local-storage class as default for simplicity.
+	// We set standard class as default for simplicity.
 	// The default storage class of kind is local-path-provisioner which
 	// consumes local storage like local-volume-provisioner. However, it's not
 	// stable in our e2e testing.
 	if framework.TestContext.Provider == "gke" || framework.TestContext.Provider == "aws" || framework.TestContext.Provider == "kind" {
```

Comment:
Should we just remove the `kind` condition here instead of changing the `local-storage` because from the comment above, we want to use local-storage at least on GKE and EKS.

Reply:
I have already remove localpv deployment in hack/e2e.sh